packages/react-vapor/src/components/input/tests/InputConnected.spec.tsx

```diff
@@ -172,5 +175,18 @@ describe('<InputConnected />', () => {
             expect(validate(newInputState.value)).toBe(false);
             expect(newInputState.valid).toBe(validate(newInputState.value));
         });
+        it('should call changeToDirty is set as props', () => {
+            const changeToDirtySpy = jasmine.createSpy();
+            const wrapperInputConnected = shallowWithState(<InputConnected />, {});

```

Comment:
I think you can use the .dive() on shallowWithState
```
wrapperInputConnected.find(input).simulate('onchange');
```
and validate if the props is called with the full loop with the input element